src/main/java/hudson/plugins/claim/UserClaimsAction.java

```diff
@@ -0,0 +1,63 @@
+package hudson.plugins.claim;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+
+import javax.annotation.Nonnull;
+
+import hudson.Extension;
+import hudson.model.Action;
+import hudson.model.ModelObject;
+import hudson.model.TransientUserActionFactory;
+import hudson.model.User;
+
+public class UserClaimsAction extends AbstractAssignedClaimsReport {
+
+	/**
+	 * Add the {@link UserClaimsAction} to all {@link User} instances.
+	 */
+	@Extension(ordinal = -1000)
+	public static class TransientUserClaimsActionactoryImpl extends TransientUserActionFactory {
+		/**
+		 * {@inheritDoc}
+		 */
+		@Override
+		public Collection<? extends Action> createFor(User target) {
+			return Collections.singleton(new UserClaimsAction(target));
+		}
+	}
+	
+	@Nonnull
+	private final User targetUser;
+
+	public UserClaimsAction(@Nonnull User user) {
+		targetUser = user;
+	}
+	
+	@Override
+	public ModelObject getOwner() {
+		return targetUser;
+	}
+
+
+	@Override
+	protected boolean isDisplayed(AbstractClaimBuildAction<?> claimAction) {
+		return super.isDisplayed(claimAction) && claimAction.getClaimedBy().equals(targetUser.getId());
```

Comment:
I was able to get empty reports for a user with actual claims under an instance run by `hpi:run`, with no security configured.
I suspect this check is the culprit, and should probably compare actual `User` objects, taken from `User.get(...)`
I believe at some point, I'll have to take a look at the actual data stored as claimed and assigned data.

Reply:
Can you provide some details on how to reproduce? We have a custom build with these changes running since more than two weeks on one server and we haven't noticed any problem until now.

Reply:
I just ran under hpi:run, with no auth or user creation control activated. 
I created another user than the admin one, 
and using the admin one, assigned this user a test
Then I went to the People tab, click on the user, then on his Claims report
No tests were shown.

I believe this is a corner case, and that MOST of the time, this is working under standard scenarios, but I would prefer that Users are compared in a consistent way in the plugin.

Reply:
Can you detail a bit more what you did? 
When i start with `hpi:run`, I have no user at all (neither admin nor anymone else). The only way I've seen to create a user is to use the script console and to execute something like `User.get("foo25", true, [:])` but in this case I can't assign anything to this user, probably because there is still no current user.

Reply:
you might be right, I might have a specific configuration that gets reused when running hpi:run as I also have some jobs that I preconfigured.
I think the key might be to configure jenkins security. I have these settings on this instance, could you check  if it would allow you to get the same behavior as I have?
![image](https://user-images.githubusercontent.com/8553578/101821142-da781c00-3b27-11eb-9b97-c1818bf47ddc.png)

Reply:
I managed to debug the issue.

Under these settings, I am able to register a new user, and I make sure the id has uppercase characters by entering every data in upper case.
The user will then have an uppercased id in Jenkins memory, and be serialized in the user database with this uppercased id.
Claiming a test then registers the same uppercased id as the claiming user.

Restarting jenkins will have it reload its user database, and under the same settings, it will use the `CaseInsensitive` `IdStrategy`, which will lower-case the ids on load to make future comparisons faster.
At this point, using the provided code, comparing the users will lead to a case-sensitive comparison which will obviously consider both ids different.

I just pushed a fix.